rotkehlchen/accounting/cost_basis/prefork.py

```diff
@@ -46,20 +46,27 @@ def handle_prefork_asset_acquisitions(
 
     events = []
     for acquisition in acquisitions:
+        asset = acquisition[0]
+        cost_basis.obtain_asset(
+            amount=amount,
+            timestamp=timestamp,
+            rate=price,
+            index=starting_index,
+            asset=asset,
+        )
         event = ProcessedAccountingEvent(
             type=AccountingEventType.PREFORK_ACQUISITION,
             notes=acquisition[1],
             location=location,
             timestamp=timestamp,
-            asset=acquisition[0],
-            taxable_amount=amount,
-            free_amount=ZERO,
+            asset=asset,
+            taxable_amount=ZERO,
+            free_amount=amount,
```

Comment:
here you are swapping the taxable/free values... is this intendeed? I don't see why

Reply:
Just for displaying purposes since I thought that is more convenient to show event as tax free rather than with taxable amount but zero profit and loss

Reply:
what? I don't follow here

Reply:
Hmmm yeah I think just forget it. I think I mixed some things together